.releaserc.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
         replacements: [
           {
             files: ['smartcar/__init__.py'],
-            from: "__version__ = 'semantic-release'",
```

Comment:
@rsimari  I think [this was the issue ](https://github.com/smartcar/python-sdk/pull/85/files#diff-68d117b5dacb59a4945f71fffa6615317258a8503bf4e88d875eefd41d1d2cc1R1) . We cleaned it up with prettier to use double quotes but the release replacements still has single quotes, so it fails to find the replacements. This should allow the deploy to go through.

Reply:
:see_no_evil: